src/api/index.ts

```diff
@@ -129,7 +129,7 @@ export const getTokenBalances = async (tokenList: DefaultTokenObject[], account?
     name: token.symbol || token.name || token.address || 'Unknown Token',
     decimals: token.decimals,
     address: token.address,
-    balance: balances[i] as BigNumber,
+    balance: balances[i] || toBigNumber(0) as BigNumber,
```

Comment:
this fixes `null` `WETH` balance, though _why_ is `getETHBalance` returning `null` in the first place?

Reply:
I would somewhat understand if it was `undefined`, but `null`... I'm stumped

Reply:
me neither. but since this is happening with web3.eth.getBalance, sounds like not our issue. yay pointing fingers!